nebullvm/operations/inference_learners/neural_compressor.py

```diff
@@ -41,6 +41,8 @@ class NeuralCompressorInferenceLearner(BaseInferenceLearner, ABC):
         model (torch.fx.GraphModule): Torch fx graph model.
     """
 
+    name = "IntelNeuralCompressor"
```

Comment:
Just a question: where do we use the name?

Reply:
It is printed in the final report, in order to print the name of the technique used to generate the optimized model.